django/template/base.py

```diff
@@ -903,7 +903,12 @@ def _resolve_lookup(self, context):
                                 raise
         except Exception as e:
             template_name = getattr(context, 'template_name', 'unknown')
-            logger.debug('{} - {}'.format(template_name, e))
+            logger.debug(
+                'Exception raised while resolving variable %s in template %s',
+                bit,
```

Comment:
I'm not thrilled with relying on the forloop above to begin executing in order for `bit` to exist in locals(). If this seems like an issue I can do something a little nastier like `locals().get('bit', 'undefined')` to get the variable name